New Featured Artist: Inferi (#2372)
ppy/osu-wiki: news/2019-07-20-new-featured-artist-inferi.md

---
layout: post
title: "New Featured Artist: Inferi"
date: 2019-07-20 00:00:00 +0000
---

Born again from the flames and into the Featured Artist family, **Inferi** joins the roster!

![](https://assets.ppy.sh/artists/43/header.jpg)

Delving deeper into the pits of the highly complex and commanding sound of metal and its subgenres, **Inferi** stands atop the Featured Artist lineup as the premiere standard for intense, powerful songwriting and musical talent.

> We're happy to announce *30* fearsome tracks spanning **Inferi**'s three latest albums: From the highly praised concept album *The Path of Apotheosis*, through the nine rings of hell within *Revenant*, to the Rebirth of their 2009 album *The End of an Era*, **Inferi** is a relentless powerhouse within the technical death metal scene.

Reviewer: ```suggestion
We're happy to announce *30* fearsome tracks spanning **Inferi**'s three latest albums: from the highly praised concept album *The Path of Apotheosis*, through the nine rings of hell within *Revenant*, to the Rebirth of their 2009 album *The End of an Era*, **Inferi** is a relentless powerhouse within the technical death metal scene.
```

### The Promethean Kings

<audio controls>
    <source src="THIS IS AN EPHEMERAL LINK" type="audio/mpeg">
</audio>
 
### The Path of Apotheosis

<audio controls>
    <source src="CHECK IT OUT, MORE EPHEMERAL" type="audio/mpeg">
</audio>

### Thy Menacing Gaze

<audio controls>
    <source src="HOLY CRUMBS EPHEMERAL LINKS SOMETHING HERE" type="audio/mpeg">
</audio>

### Malevolent Sanction

<audio controls>
    <source src="WOO. EPHEMERAL." type="audio/mpeg">
</audio>

### The Warrior's Infinite Opus

<audio controls>
    <source src="MAN EPHEMERAL HAS HIS WORK CUT OUT FOR HIM" type="audio/mpeg">
</audio>

### Cursed Unholy

<audio controls>
    <source src="NEARLY DONE EPHEMERAL, LAST LINK" type="audio/mpeg">
</audio>

---------

As with every other Featured Artist release, you can find the rest of **Inferi**'s music fully pre-timed and ready to download over at their [Featured Artist page](https://osu.ppy.sh/beatmaps/artists/43). You can sleep easy knowing all of these tracks are 100% free to use for your mapping pleasure, so snag a beatmap template and get going!

> just like we've been alluding the last few newsposts, there's still a *lot* of exciting new artists hiding in the wings. Keep on the lookout for small hints and teasers as to who might be rearing up very soon because things are about to get warped.

Reviewer: ```suggestion
Just like we've been alluding the last few news posts, there's still a *lot* of exciting new artists hiding in the wings. Keep on the lookout for small hints and teasers as to who might be rearing up very soon, because things are about to get warped.
```

> Until next time!

Reviewer: since it's right above ur signature i think u can do "Until next time," ? not sure
```suggestion
Until next time,
```
